src/gui/mainwindow.cpp

```
@@ -24,7 +24,7 @@ MainWindow::MainWindow(QWidget *parent) :
     this->setWindowTitle("SQLite Query Analyzer");
     this->connectSignalSlots();
 
-    this->database = std::make_unique<Database>();
+    this->database = std::make_unique<SqliteDatabase>();
     this->analyzer = std::make_unique<DbAnalyzer>(database.get());
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)